server/api/controllers/ArticleController.js

```diff
@@ -0,0 +1,13 @@
+import models from '../../models';
+
+export const createArticle = async (req, res) => {
+  try {
+    const newArticleData = req.body;
+    const userId = req.user.id;
+    newArticleData.authorId = userId;
+    const newArticle = await models.Article.create(newArticleData);
+    res.status(201).send({ data: newArticle });
```

Comment:
@Princesso Please add a `return` statement to line `9` so that it reads ``` return res.status(201).send({ data: newArticle }); ```.

Reply:
Change effected.